crates/tabby-index/src/code/index.rs

```diff
@@ -99,7 +99,9 @@ async fn add_changed_documents(
                 continue;
             };
 
-            if cloned_index.is_indexed(&key) {
+            let id = SourceCode::to_index_id(&repository.source_id, &key).id;
```

Comment:
@xxs-wallace in last round of refactoring, id's definition is changed, but the `is_indexed` check is not updated simultaneously